.github/workflows/bootstrap.yml

```diff
@@ -67,4 +63,4 @@ jobs:
           cd /tmp
           ./install-devenv.sh  < $repo/ci/install-devenv-checks.sh
       - name: bootstrap sentry
-        run: ./ci/devenv-bootstrap.sh
+        run: ./ci/devenv-bootstrap.sh || cat /Users/runner/.colima/_lima/colima/ha.stderr.log
```

Comment:
is this needed or just for debugging?

Reply:
for debugging but i'm leaving it tehre since it's nice to have